src/pay/batch_payer.py

```diff
@@ -429,6 +429,7 @@ def attempt_single_batch(self, payment_records, op_counter, dry_run=None):
 
             # if pymnt_amnt becomes 0, don't pay
             if pymnt_amnt == 0:
+                payment_item.paid = PaymentStatus.SKIPPED
```

Comment:
I would argue DONE is better suited for this case.

Reply:
SKIPPED status is for payment items which are avoided with intention. In most cases due to the fact that costs of the transaction are higher than the reward amount. For many bakers, this transaction is not failed because it is avoided with intention and it is not DONE because the payout was not really performed by the baker, thus the idea of this new status.

Reply:
I think the new status is better after seeing your comments.